integration_tests/commands/resp/set_data_cmd_test.go

```diff
@@ -0,0 +1,168 @@
+package resp
+
+import (
```

Comment:
let me know if i am missing something here. migrating here just means "copy/pasting" the relevant test cases? and calling the helpers in resp pkg instead of async pkg.?

Reply:
Yes, it mostly means copy pasting the test-cases to the resp package and making sure they pass the tests.